visualization/app/codeCharta/util/gameObjectsParser/gameObjectsSchema.json

```diff
@@ -0,0 +1,271 @@
+				"anyOf": [
+					{
+						"$id": "#/properties/gameObjectPositions/items/anyOf/0",
+						"type": "object",
+						"title": "The first anyOf schema",
+						"description": "An explanation about the purpose of this instance.",
+						"default": {},
+						"examples": [
+							{
+								"name": "root.analysis.import.SourceCodeParser.src.test.kotlin.de.maibornwolff.codecharta.importer.sourcecodeparser.oop.core.extract.HelloWorldTest",
+								"position": {
+									"x": 69.0,
+									"y": 18.5,
+									"z": 228.5
+								},
+								"scale": {
+									"x": 9.0,
+									"y": 5.0,
+									"z": 9.0
+								}
+							}
+						],
+						"required": ["name", "position", "scale"],
+						"properties": {
+							"name": {
+								"$id": "#/properties/gameObjectPositions/items/anyOf/0/properties/name",
+								"type": "string",
+								"title": "The name schema",
+								"description": "An explanation about the purpose of this instance.",
+								"default": "",
+								"examples": [
+									"root.analysis.import.SourceCodeParser.src.test.kotlin.de.maibornwolff.codecharta.importer.sourcecodeparser.oop.core.extract.HelloWorldTest"
+								]
+							},
+							"position": {
+								"$id": "#/properties/gameObjectPositions/items/anyOf/0/properties/position",
+								"type": "object",
+								"title": "The position schema",
+								"description": "An explanation about the purpose of this instance.",
+								"default": {},
+								"examples": [
+									{
+										"x": 69.0,
+										"y": 18.5,
+										"z": 228.5
+									}
+								],
+								"required": ["x", "y", "z"],
+								"properties": {
+									"x": {
+										"$id": "#/properties/gameObjectPositions/items/anyOf/0/properties/position/properties/x",
+										"type": "number",
+										"title": "The x schema",
+										"description": "An explanation about the purpose of this instance.",
+										"default": 0.0,
+										"examples": [69.0]
+									},
+									"y": {
+										"$id": "#/properties/gameObjectPositions/items/anyOf/0/properties/position/properties/y",
+										"type": "number",
+										"title": "The y schema",
+										"description": "An explanation about the purpose of this instance.",
+										"default": 0.0,
+										"examples": [18.5]
+									},
+									"z": {
+										"$id": "#/properties/gameObjectPositions/items/anyOf/0/properties/position/properties/z",
+										"type": "number",
+										"title": "The z schema",
+										"description": "An explanation about the purpose of this instance.",
+										"default": 0.0,
+										"examples": [228.5]
+									}
+								},
+								"additionalProperties": true
+							},
+							"scale": {
+								"$id": "#/properties/gameObjectPositions/items/anyOf/0/properties/scale",
+								"type": "object",
+								"title": "The scale schema",
+								"description": "An explanation about the purpose of this instance.",
```

Comment:
I am not sure about the purpose of the descriptions. Should we adjust them?

Reply:
Hm, it doesn't make a huge difference, but better have a descriptive description. I will adjust it.